Helper/Data.php

```diff
@@ -672,12 +674,16 @@ public function getAPIKey($storeId = null)
      */
     public function getClientKey($storeId = null)
     {
-        return trim(
-            $this->getAdyenAbstractConfigData(
-                $this->isDemoMode($storeId) ? 'client_key_test' : 'client_key_live',
-                $storeId
-            )
+        $clientKey = $this->getAdyenAbstractConfigData(
+            $this->isDemoMode($storeId) ? 'client_key_test' : 'client_key_live',
+            $storeId
         );
+
+        if (is_null($clientKey)) {
+            return false;
```

Comment:
Let's turn `string` configurations into `string|null` to represent lack of setup:
```suggestion
            return null;
```